### ISSUE TITLE
Rule Proposal："<strong>" and "<em>" tags should be used

### ISSUE THREAD

**Issue by snowbabykang** [vuejs/eslint-plugin-vue#1475]
**Please describe what the rule should do:**

I want to add aria-hidden to the  <i>；  or replace <i> with <em>

**What category should the rule belong to?**

[ X] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[X ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<i class="icon-delete"></i>
<b>test</b>
```

**Additional context**

Our company uses Sonar to monitor code quality，sonar always tells us the following
 
![image](https://user-images.githubusercontent.com/12569179/114364337-e0503d80-9bab-11eb-8a24-e84be2543a44.png)

**Comment by ota-meshi** [comment]
I don't know why this rule is needed. Can you explain?

**Comment by snowbabykang** [comment]
> I don't know why this rule is needed. Can you explain?

in order to convey semantics；
This rule is relaxed in case of icon fonts usage.  can add attribute [aria-hidden="true"]

**Comment by ota-meshi** [comment]
I'm not familiar with that semantics. Could you explain the details?

**Comment by snowbabykang** [comment]
> I'm not familiar with that semantics. Could you explain the details?

Well, it's only because Sonar hint it, and I thought it made sense, so, I thought there was a rule that would tip me off；

And Sonar explain:
The \<strong>/\<b> and \<em>/\<i> tags have exactly the same effect in most web browsers, but there is a fundamental difference between them: \<strong> and \<em> have a semantic meaning whereas \<b> and \<i> only convey styling information like CSS.

While \<b> can have simply no effect on a some devices with limited display or when a screen reader software is used by a blind person, \<strong> will:

Display the text bold in normal browsers
Speak with lower tone when using a screen reader such as Jaws
Consequently:

in order to convey semantics, the \<b> and \<i> tags shall never be used,
in order to convey styling information, the \<b> and \<i> should be avoided and CSS should be used instead.

**Comment by ota-meshi** [comment]
Thank you for your comment.
Perhaps I understand that the rule is not just a rule that reports `<b>` and `<i>`.
I find it difficult to implement this if someone with a detailed knowledge of accessibility does not implement it.
How about making a suggestion with eslint-plugin-vuejs-accessibility? I think they are much more familiar with it than I am.
https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility

**Comment by snowbabykang** [comment]
> Thank you for your comment.
> Perhaps I understand that the rule is not just a rule that reports `<b>` and `<i>`.
> I find it difficult to implement this if someone with a detailed knowledge of accessibility does not implement it.
> How about you suggesting this rule with eslint-plugin-vuejs-accessibility? I think they are much more familiar with it than I am.
> https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility

Thanks for your advice

**Comment by ota-meshi** [comment]
It seems that eslint-plugin-vuejs-accessibility suggested this rule, so I'll close this issue.
